app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from typing import List
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
+    vaccinated_friends = 0
```

Comment:
Why you need this variable?

Reply:
At first I though I needed it to keep track of friends, but now I see that it is indeed useless